frontend/src/components/profileBanner/index.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react'
+import styles from './profileBanner.scss'
+import { inject, observer } from 'mobx-react'
+import { Icon, Cropper } from '..'
+
+@inject('store') @observer
+class ProfileBanner extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      upBanner: null,
+      draggingOverBanner: false,
+    }
+  }
+
+  onEditBanner = (input) => {
+    this.handleInput(input, 'upBanner')
+    this.onBannerDragLeave()
+  }
+
+  banUser() {
+    this.setState({ banUser: true })
+  }
+
+  handleInput = (input, stateVar) => {
+    input.value = ''
+
+    if (input.target.files && input.target.files.length > 0) {
```

Comment:
Might want to use optional chaining here to make sure you don't get an nullpointerexception. Also removes the necessity for the &&.

```suggestion
    if (input?.target?.files?.length > 0) {
```

Reply:
Since I see that you use this multiple times I'd even say make something like this under line 27:

`const files = input?.target?.files`

If one of the properties does not exist files will be undefined